Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -467,6 +470,45 @@ def main():  # pragma: no cover
         elif command == 'core-get-incidents':
             return_outputs(*get_incidents_command(client, args))
 
+        elif command == 'core-terminate-process':
+            if not is_demisto_version_ge(version=TERMINATE_BUILD_NUM,
+                                         build_number=TERMINATE_SERVER_VERSION):
+                return 'This command is only available for XSIAM 2.4'
```

Comment:
what happens if we run the command in a lower version than the required one? If it failed with specific error, we might want to catch it in the exception instead checking specific build number

Reply:
there isn't specific error :(